src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleNoTrailingComma.java

```diff
@@ -21,15 +21,15 @@ public String toString() {
       };
   }
   
-  @Test2(value={(false) ? "" : "foo"}, more={(true) ? "" : "bar"})
+  @Test2(value={"foo"}, more={"bar"})
```

Comment:
@romani I don't agree with changing our inputs just to satisfy eclipse's issue which isn't even an issue for me.
These are weird cases and we should keep them.

Reply:
I added such cases to non-compilable.

>just to satisfy eclipse's issue which isn't even an issue for me.

After import to IDE, there should be no ERRORs. No excuse for this simple rule. 
Ideally there should be no warnings, but we can ease this rule for now, as  it does not "block" UT execution.

Without such simple principle, it will hard to attract new contributors from aside.
Entrance threshold to project should as minimal as possible to ease start of contributions.